mingw-w64-procs/PKGBUILD

```
@@ -0,0 +1,51 @@
+# Maintainer: Maksim Bondarenkov <maksapple2306@gmail.com>
+
+_realname=procs
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=0.14.5
+pkgrel=1
+pkgdesc="A modern replacement for ps written in Rust (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url='https://github.com/dalance/procs'
+license=('spdx:MIT')
+makedepends=("${MINGW_PACKAGE_PREFIX}-rust")
+options=('!lto')
+source=("https://github.com/dalance/procs/archive/v${pkgver}/${_realname}-${pkgver}.tar.gz")
+sha256sums=('539b88565c775a106063da5cc5148cfdc7e010534f3dbc90cb8f6317d51ca96b')
+
+prepare() {
+  cd "${srcdir}/${_realname}-${pkgver}"
+
+  cargo fetch --locked --target "$(rustc -vV | sed -n 's/host: //p')"
+}
+
+build() {
+  cd "${srcdir}/${_realname}-${pkgver}"
+
+  cargo build --release --frozen
+}
+
+check() {
+  cd "${srcdir}/${_realname}-${pkgver}"
+
+  cargo test --release --frozen
+}
+
+package() {
+  cd "${srcdir}/${_realname}-${pkgver}"
+
+  cargo install \
+    --offline \
+    --no-track \
+    --frozen \
+    --path . \
+    --root "${pkgdir}${MINGW_PREFIX}"
+
+  local _complete="./target/release/procs --gen-completion"
```

```suggestion
  local _complete="./target/release/procs --gen-completion-out"
```

The generated shell completions are wrong. You can also simplify those shell completions install command with the above command `$_complete > path_goes_here` (all those pipe and stdin can be replaced).

---

oops, I received a error from fish while `procs --help`

---

fixed, thanks again